<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventDialog</class>
 <widget class="QDialog" name="EventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EVENT PROPERTY</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>870</width>
     <height>590</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="buttonEventTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>200</height>
     </size>
    </property>
    <property name="text">
     <string>Event
Time: 15/30sec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNatAvailable">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>200</height>
     </size>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGsensor">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>200</height>
     </size>
    </property>
    <property name="text">
     <string>G-Sensor
Record: Off/On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTrigger">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>200</height>
     </size>
    </property>
    <property name="text">
     <string>Trigger
Record: Off/On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClose">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>200</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSave">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>200</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
